seatunnel-apis/seatunnel-api-flink/src/main/java/org/apache/seatunnel/flink/FlinkEnvironment.java

```diff
@@ -322,4 +326,61 @@ private void setCheckpoint() {
         }
     }
 
+    public StreamExecutionEnvironment creatMetricStreamEEnvironment() {
```

Comment:
```suggestion
    public StreamExecutionEnvironment creatStreamEnvironment() {
```
Not only have metrics config will run this method.